components/accept-financial-contributions/AcceptContributionsMyself.js

```diff
@@ -0,0 +1,393 @@
+    router: PropTypes.object,
+    LoggedInUser: PropTypes.object.isRequired,
+    refetchLoggedInUser: PropTypes.func.isRequired,
+    addBankAccount: PropTypes.func,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: null,
+    };
+
+    this.messages = defineMessages({
+      header: {
+        id: 'acceptContributions.picker.header',
+        defaultMessage: 'Accept financial contributions',
+      },
+      howDoesItWork: {
+        id: 'paymentMethods.manual.HowDoesItWork',
+        defaultMessage: 'How does it work?',
+      },
+      howDoesItWorkDetails: {
+        id: 'acceptContributions.HowDoesItWork.details',
+        defaultMessage:
+          'Financial contributors will be able to choose "Bank transfer" as a payment method. Instructions to make the transfer, which you define, will be emailed to them, along with a unique order ID. Once you receive the money, you can mark the corresponding pending order as paid and the funds will be credited to the Collective\'s balance.',
+      },
+      definePaymentInstructions: {
+        id: 'acceptContributions.definePaymentInstructions',
+        defaultMessage: 'Define payment instructions',
+      },
+      definePaymentInstructionsDetails: {
+        id: 'acceptContributions.definePaymentInstructionsDetails',
+        defaultMessage:
+          'Include any details contributors will need to send you money, such as: account name and number; IBAN, Swift, or routing codes; bank name and address, etc. The amount and order ID will be automatically included.',
+      },
+      connectStripe: {
+        id: 'collective.connectedAccounts.stripe.button',
+        defaultMessage: 'Connect Stripe',
+      },
+      cancel: {
+        id: 'actions.cancel',
+        defaultMessage: 'Cancel',
+      },
+      save: {
+        id: 'save',
+        defaultMessage: 'Save',
+      },
+      transactionFee: {
+        id: 'acceptContributions.transactionFee',
+        defaultMessage: '5% transaction fee applies',
+      },
+      addBankAccount: {
+        id: 'acceptContributions.addBankAccount',
+        defaultMessage: 'Add bank account',
+      },
+      bankAccountUpgradeInfo: {
+        id: 'acceptContributions.bankAccountUpgradeInfo',
+        defaultMessage: "Your first $1000 is free, then you'll need to upgrade to a paid plan",
+      },
+    });
+  }
+
+  connectStripe = () => {
+    const { collective } = this.props;
+    const service = 'stripe';
+
+    connectAccount(collective.id, service)
+      .then(json => {
+        return (window.location.href = json.redirectUrl);
+      })
+      .catch(err => {
+        console.error(`>>> /api/connected-accounts/${service} error`, err);
+      });
+  };
+
+  submitBankAccountInformation = async bankAccountInfo => {
+    // set state to loading
+    this.setState({ loading: true });
+
+    // prepare objects
+    const account = {
+      legacyId: this.props.LoggedInUser.CollectiveId,
```

Comment:
This is not the account we want to add the bank account information.

It should be the collective passed in props.

Reply:
Forget that.